doc/source/whatsnew/v1.1.0.rst

```diff
@@ -379,7 +379,7 @@ Other API changes
 - ``loc`` lookups with an object-dtype :class:`Index` and an integer key will now raise ``KeyError`` instead of ``TypeError`` when key is missing (:issue:`31905`)
 - Using a :func:`pandas.api.indexers.BaseIndexer` with ``count``, ``min``, ``max``, ``median``, ``skew``,  ``cov``, ``corr`` will now return correct results for any monotonic :func:`pandas.api.indexers.BaseIndexer` descendant (:issue:`32865`)
 - Added a :func:`pandas.api.indexers.FixedForwardWindowIndexer` class to support forward-looking windows during ``rolling`` operations.
--
+- Removed ``private_key`` and ``verbose`` from gbq (:issue:`34654`)
```

Comment:
can you co-locate this with the other issue where we removed a keyword from gbq; also pls use the references to read_gbq (see the other issue)